cinema/models.py

```diff
@@ -1,10 +1,36 @@
 from django.db import models
 
 
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.PositiveIntegerField()
+    seats_in_row = models.PositiveIntegerField()
+
+    def __str__(self):
+        return self.name
+
+
 class Movie(models.Model):
```

Comment:
The class definition for `Movie` is missing a commentable line marker. Ensure it follows the pattern `/++line_number++/` for consistency with other class definitions.